pokemongo_bot/cell_workers/pokemon_catch_worker.py

```diff
@@ -33,7 +33,9 @@ def work(self):
                 if 'status' in response_dict['responses']['ENCOUNTER']:
                     if response_dict['responses']['ENCOUNTER']['status'] is 7:
                         logger.log('[x] Pokemon Bag is full!', 'red')
-                        self.bot.initial_transfer()
+                        worker = InitialTransferWorker(self.bot)
```

Comment:
what about implementing a return code for this case that you can handle in the if-block at around line 69 in `pokemongo_bot/__init__.py`?